documentation.html

```diff
@@ -0,0 +1,230 @@
+<!doctype html><html lang=en><meta charset=utf-8><meta http-equiv=x-ua-compatible content="ie=edge"><meta name=viewport content="width=device-width,initial-scale=1"><title>GH_ROBBER</title><style media=screen>/*!* Bootstrap v3.3.7 (http://getbootstrap.com)
```

Comment:
Waht is this file?

Reply:
API documentation. Can be downloaded and opened in a browser. Unfortunately, this is the only option I found to save the postman documentation.